go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/sirupsen/logrus v1.9.0
 	github.com/spf13/cast v1.5.0
 	github.com/spf13/cobra v1.7.0
-	github.com/spf13/pflag v1.0.6-0.20200504143853-81378bbcd8a1
+	github.com/spf13/pflag v1.0.5
```

Comment:
We don't want to downgrade the pflag dependency, see 97ad7eb549c1820958d58ad6f643b625acebddc9.